src/vcpkg/commands.dependinfo.cpp

```diff
@@ -197,14 +196,13 @@ namespace vcpkg::Commands::DependInfo
 
     std::string create_dgml_as_string(const std::vector<PackageDependInfo>& depend_info)
     {
-        std::string s;
-        s.append("<?xml version=\"1.0\" encoding=\"utf-8\"?>");
-        s.append("<DirectedGraph xmlns=\"http://schemas.microsoft.com/vs/2009/dgml\">");
+        std::string s =
```

Comment:
Separating the declaration was intentional. I think if this is being touched it should be changed to use the real XML writer instead.

Reply:
Done